### PR TITLE
Docs/add blog documentation for marketing

### PR DESCRIPTION
**Pull request by omonk**

### CODE REVIEW

File: docs/blog-marketing.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The automated blog workflow has been designed to maintain the same workflow as any usual medium publication but we’ve added two small (but important) steps.
+
+Standard Workflow:
+
+- Author writes blog post and submits it for review to the YLD medium publication
+- Marketing will proof read and request any peer reviews required for the topic
+- Once the post has been approved, YLD publishes the blog normally.
+
+New additional steps
```
[the remark this answers is not in the document]
Comment (jeysal) [reply docs/blog-marketing.md]: Gracefully may mean don't publish before it is done, publish it with what we have, etc. depending on requirements

File: docs/blog-marketing.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- Once you’re in editing mode click the three dots on the top right to open the dropdown. At the bottom of the dropdown list you’ll want to click `Customize canonical link`.
+
+  ![edit canonical](./assets/canonical.png)
+
+- Add the new blog post link from the yld.io site, hit save and you’re good to go!
+
+  ![edit canonical](./assets/canonical-url.png)
+
+## Add preivew image
```
Comment (jeysal) [inline docs/blog-marketing.md]:
```suggestion
## Add preview image
```

File: docs/blog-marketing.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+This step is only required if the blog published on medium has no images. Having images in blog posts is not critical to the new content but has two but It is advised that you encourage authors to add content but if not possible then these are the steps to follow to include the image.
```
Comment (jeysal) [inline docs/blog-marketing.md]: this sentence :fearful: 
I think you have some leftover words you wanted to delete :smile:

File: docs/blog-marketing.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- Add the new header image
+- Click publish and you're done!
+
+## FAQ
+
+Q: If I change content on medium will it update on yld.io?
+A: No, we only look for new urls in the RSS content and compare it to blog posts already published it on the CMS.
+
+Q: How do I change content on a blog post then? Do I have to update it twice?
+A: Yes, for this reason we encourage authors and proof readers to make sure everything is 100% ready before publishing but it can be easy to miss things sometimes! To change blog post content on CMS you'll need to write it in [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). Try not to change too much!
```
Comment (jeysal) [inline docs/blog-marketing.md]: Is deleting the post and triggering a re-export somehow an option?

Comment (omonk) [reply docs/blog-marketing.md]: Yes definitely an option but would require manual triggering, will include

File: docs/blog-marketing.md (hunk 3 above)
Comment (omonk) [reply docs/blog-marketing.md]: 😱

**Pull request merged**